gio/src/error.rs

```diff
@@ -1,5 +1,7 @@
 // Take a look at the license at the top of the repository in the LICENSE file.
 
+use crate::glib::translate::from_glib;
```

Comment:
```suggestion
use crate::glib::translate::*;
```
you need into_glib as well, also those two new imports needs a version guard